src/components/markdown.js

```diff
@@ -55,7 +55,7 @@ const Markdown = (props) => {
     const GithubStats = (props) => {
         let link = "https://github-readme-stats.vercel.app/api?username=" + props.github + "&show_icons=true"
         if (props.show) {
-            return (<>{`<img align="center" src="${link}" alt="${props.github}" />`}<br /><br /></>);
```

Comment:
`align="center"` in this line is there for a reason. Don't change this please

Reply:
Brought back `align="center"` with the recent commit